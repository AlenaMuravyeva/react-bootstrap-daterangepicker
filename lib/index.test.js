import React from 'react';
import { mount } from 'enzyme';
import DateRangePicker from './index.js';

const delay = async (ms) => new Promise(r => setTimeout(r, ms));

test('DateRangePicker - simple', async () => {
  // Render a checkbox with label in the document
  const picker = mount(<DateRangePicker>click me</DateRangePicker>);
  expect(picker.html()).toMatchSnapshot();
  //await delay(100);
  expect(document.body).toMatchSnapshot();
});
