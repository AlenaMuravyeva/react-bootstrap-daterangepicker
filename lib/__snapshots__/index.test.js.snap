// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateRangePicker - simple 1`] = `"<div>click me</div>"`;

exports[`DateRangePicker - simple 2`] = `
<body>
  <div
    class="daterangepicker dropdown-menu ltr show-calendar opensright"
  >
    <div
      class="calendar left"
    >
      <div
        class="daterangepicker_input"
      >
        <input
          class="input-mini form-control"
          name="daterangepicker_start"
          type="text"
          value=""
        />
        <i
          class="fa fa-calendar glyphicon glyphicon-calendar"
        />
        <div
          class="calendar-time"
          style="display: none;"
        >
          <div />
          <i
            class="fa fa-clock-o glyphicon glyphicon-time"
          />
        </div>
      </div>
      <div
        class="calendar-table"
      />
    </div>
    <div
      class="calendar right"
    >
      <div
        class="daterangepicker_input"
      >
        <input
          class="input-mini form-control"
          name="daterangepicker_end"
          type="text"
          value=""
        />
        <i
          class="fa fa-calendar glyphicon glyphicon-calendar"
        />
        <div
          class="calendar-time"
          style="display: none;"
        >
          <div />
          <i
            class="fa fa-clock-o glyphicon glyphicon-time"
          />
        </div>
      </div>
      <div
        class="calendar-table"
      />
    </div>
    <div
      class="ranges"
    >
      <div
        class="range_inputs"
      >
        <button
          class="applyBtn btn btn-sm btn-success"
          disabled="disabled"
          type="button"
        >
          Apply
        </button>
         
        <button
          class="cancelBtn btn btn-sm btn-default"
          type="button"
        >
          Cancel
        </button>
      </div>
    </div>
  </div>
</body>
`;
